evm/src/libraries/WormholeCctpMessages.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.22;
+
+import {IWormhole} from "src/interfaces/IWormhole.sol";
+
+import {BytesParsing} from "src/libraries/BytesParsing.sol";
+import {Utils} from "src/libraries/Utils.sol";
+
+library WormholeCctpMessages {
+    using Utils for address;
+    using BytesParsing for bytes;
+
+    // Payload IDs.
+    uint8 private constant DEPOSIT = 0x1;
+
+    uint256 public constant PAYLOAD_OFFSET = 149;
```

Comment:
Doesn't seem to be used anywhere?

Reply:
Removed here: https://github.com/wormhole-foundation/wormhole-circle-integration/pull/48/commits/da6a7e324997c6ed24f6e1d1f6dd569e28405fc7